tethys_services/backends/hs_restclient_helper.py

```diff
@@ -0,0 +1,29 @@
+import hs_restclient as hs_r
+from django.conf import settings
+
+
+def get_oauth_hs(request):
+    hs = None
+    error_msg_head = "Failed to initialize hs object: "
+
+    if request.user.social_auth.count() == 0:
+        raise Exception(error_msg_head + "Not logged in through OAuth")
+    if request.user.social_auth.count() > 1:
```

Comment:
That's a good question. Ideally, the user is presented with a dialog and able to choose somehow. But in the case that that isn't an option, then you need to decide on a default behavior where one takes priority over the other. I don't really have a preference on that, so I'll let you decide if it comes to it.